dev-docs/RFCs/v4.0.0/names_for_concepts_types_functions.md

```diff
@@ -52,7 +52,7 @@ The following technical terms should be used in the documentation, the H3 codeba
 - **grid**:
     - the graph with nodes corresponding to H3 cells, and edges given by pairs of adjacent cells
     - for example, `gridDistance` is the minimal number of edges in a graph path connecting two cells
-- **point**:
+- **lat/lng point**:
```

Comment:
We should update the terminology page in the website docs to indicate that we use "lng" for "longitude" when abbreviating. Similarly I think "lat/lng point" should be on that page.

Reply:
updated in the docs!